### PR TITLE
WINUI-3339-Customization-AvatarView

### PR DESCRIPTION
**Pull request by MadhumithanJayapathi**
### Description
Addition of Customization Page for Avatar view UserGuide(UG).

### CODE REVIEW

File: winui/Avatar-view/customization.md [hunk 1]
```diff
@@ -1,10 +1,149 @@
...
 
-# Customization in WinUI AvatarView 
\ No newline at end of file
+# Customization in WinUI AvatarView 
+
+The `AvatarView` control offers a range of customization options, including the ability to modify its colors, background, and more. You can utilize the following properties to customize this control:
+
+## Border
+
+Personalize the border of `AvatarView` control by adjusting the border brush and border thickness properties to achieve the desired look.
```
Comment (Manivannan-E) [inline winui/Avatar-view/customization.md:16]: @MadhumithanJayapathi Customize the border of `AvatarView` control by using the BorderBrush and BorderThickness

File: winui/Avatar-view/customization.md [hunk 2]
```diff
@@ -1,10 +1,149 @@
...
+avatarView.CornerRadius = new CornerRadius(50);
+avatarView.BorderBrush = new SolidColorBrush(Colors.Black);
+
+{% endhighlight %}
+{% endtabs %}
+
+![WinUI AvatarView control with custom BorderThickness](avatarview_images/winui_borderthickness_avatarview.png)
+
+### Background
+
```
Comment (Manivannan-E) [inline winui/Avatar-view/customization.md:78]: @MadhumithanJayapathi ## Background

File: winui/Avatar-view/customization.md [hunk 3]
```diff
@@ -1,10 +1,149 @@
...
+avatarView.InitialsType = AvatarInitialsType.DoubleCharacter;
+avatarView.Background = new SolidColorBrush(Colors.Bisque);
+
+{% endhighlight %}
+{% endtabs %}
+
+![WinUI AvatarView control with custom background](avatarview_images/winui_double_character_initialstype_avatarview.png)
+
+### Gradients
+
```
Comment (Manivannan-E) [inline winui/Avatar-view/customization.md:106]: @MadhumithanJayapathi ## Gradient background

File: winui/Avatar-view/customization.md [hunk 4]
```diff
@@ -1,10 +1,149 @@
 ---
 layout: post
 title: Customization in WinUI AvatarView control | Syncfusion
-description: Learn all about the customization features such as Height, Width, BorderBrush, Background, FontSize and CornerRadius in WinUI AvatarView control here.
+description: Learn all about the customization features such as BorderThickness, BorderBrush, Background, FontSize and GradientBackground in WinUI AvatarView control here.
 platform: WinUI
```
[the remark this answers is not in the document]
Comment (MadhumithanJayapathi) [reply winui/Avatar-view/customization.md:5]: @Manivannan-E Changed the description as mentioned.

File: winui/Avatar-view/customization.md (hunk 1 above)
Comment (MadhumithanJayapathi) [reply winui/Avatar-view/customization.md]: @Manivannan-E Changed the Content of Border as mentioned.

File: winui/Avatar-view/customization.md (hunk 2 above)
Comment (MadhumithanJayapathi) [reply winui/Avatar-view/customization.md:73]: @Manivannan-E Changed the ###Background to ##Background.

File: winui/Avatar-view/customization.md (hunk 3 above)
Comment (MadhumithanJayapathi) [reply winui/Avatar-view/customization.md:102]: @Manivannan-E Changed the ### Gradient to ##Gradient

File: winui/Avatar-view/customization.md [hunk 5]
```diff
@@ -1,10 +1,176 @@
...
+
+{% endhighlight %}
+{% endtabs %}
+
+![WinUI AvatarView control with custom background](avatarview_images/winui_double_character_initialstype_avatarview.png)
+
+## Gradients
+
+You can use the `LinearGradientBrush` or `RadialGradientBrush` in the `Background` property to specify a range of colors for your AvatarView, as shown in the following code example.
+
```
Comment (Manivannan-E) [inline winui/Avatar-view/customization.md:104]: @MadhumithanJayapathi You can use the `LinearGradientBrush` or `RadialGradientBrush` for the `Background` property to specify a range of colors to apply the gradient background for the AvatarView,

File: winui/Avatar-view/customization.md [hunk 6]
```diff
@@ -1,10 +1,176 @@
...
+{% endhighlight %}
+{% endtabs %}
+
+![WinUI AvatarView control with Gradient Background](avatarview_images/winui_avatarview_gradient.png)
+
+## Font
+
+Customize the Font of AvatarView control by using the `FontFamily` and `Foreground` properties to achieve the desired look.
+
```
Comment (Manivannan-E) [inline winui/Avatar-view/customization.md:150]: @MadhumithanJayapathi You can customize the font of initials in the AvatarView control by using the `FontFamily` and `Foreground` properties.

File: winui/Avatar-view/customization.md (hunk 5 above)
Comment (MadhumithanJayapathi) [reply winui/Avatar-view/customization.md:104]: @Manivannan-E Changed the content as given.

File: winui/Avatar-view/customization.md (hunk 6 above)
Comment (MadhumithanJayapathi) [reply winui/Avatar-view/customization.md:150]: @Manivannan-E  Changed the content of font as given.

**Review by Manivannan-E — APPROVED** [review]
Changes are fine.

File: winui/Avatar-view/customization.md [hunk 7]
```diff
@@ -1,10 +1,176 @@
 ---
 layout: post
 title: Customization in WinUI AvatarView control | Syncfusion
-description: Learn all about the customization features such as Height, Width, BorderBrush, Background, FontSize and CornerRadius in WinUI AvatarView control here.
+description: Learn all about the various customization features available in the WinUI AvatarView control, including border, background, gradient, font, and more.
```
Comment (Chrispineagunja) [inline winui/Avatar-view/customization.md:4]: Description: Learn about the various customization features in the WinUI AvatarView control, including border, background, gradient, font, and more.

File: winui/Avatar-view/customization.md [hunk 8]
```diff
@@ -1,10 +1,176 @@
...
 platform: WinUI
 control: AvatarView
 documentation: ug
 ---
 
-# Customization in WinUI AvatarView 
\ No newline at end of file
+# Customization in WinUI AvatarView 
+
+The `AvatarView` control offers a range of customization options, including the ability to modify its border, background, and more.
```
Comment (Chrispineagunja) [inline winui/Avatar-view/customization.md:12]: The `AvatarView` control offers a range of customization options, including modifying its border, background, and more.

File: winui/Avatar-view/customization.md [hunk 9]
```diff
@@ -1,10 +1,176 @@
...
+
+## Border
+
+Customize the border of `AvatarView` control by using the `BorderBrush` and `BorderThickness` properties.
```
Comment (Chrispineagunja) [inline winui/Avatar-view/customization.md:16]: Customize the `AvatarView` control border using the `BorderBrush` and `BorderThickness` properties.

File: winui/Avatar-view/customization.md [hunk 10]
```diff
@@ -1,10 +1,176 @@
...
+
+**BorderBrush**
+
+You can customize the border color of the AvatarView control by using the `BorderBrush` property.
```
Comment (Chrispineagunja) [inline winui/Avatar-view/customization.md:20]: Customize the border color of the AvatarView control by using the `BorderBrush` property.

File: winui/Avatar-view/customization.md [hunk 11]
```diff
@@ -1,10 +1,176 @@
...
+avatarView.BorderBrush = new SolidColorBrush(Colors.Red);
+
+{% endhighlight %}
+{% endtabs %}
+
+![WinUI AvatarView control with custom BorderBrush](avatarview_images/winui_avatarview_borderbrush.png)
+
+**BorderThickness**
+
+You can customize the border thickness of the AvatarView control by using the `BorderThickness` property.
```
Comment (Chrispineagunja) [inline winui/Avatar-view/customization.md:47]: Customize the border thickness of the AvatarView control by using the `BorderThickness` property.

File: winui/Avatar-view/customization.md [hunk 12]
```diff
@@ -1,10 +1,176 @@
...
+avatarView.BorderBrush = new SolidColorBrush(Colors.Black);
+
+{% endhighlight %}
+{% endtabs %}
+
+![WinUI AvatarView control with custom BorderThickness](avatarview_images/winui_avatarview_borderthickness.png)
+
+## Background
+
+You can customize the background of the AvatarView control by using the `Background` property.
```
Comment (Chrispineagunja) [inline winui/Avatar-view/customization.md:74]: Customize the background of the AvatarView control by using the `Background` property.

File: winui/Avatar-view/customization.md [hunk 13]
```diff
@@ -1,10 +1,176 @@
...
+avatarView.Background = new SolidColorBrush(Colors.Bisque);
+
+{% endhighlight %}
+{% endtabs %}
+
+![WinUI AvatarView control with custom background](avatarview_images/winui_double_character_initialstype_avatarview.png)
+
+## Gradient background
+
+You can use the `LinearGradientBrush` or `RadialGradientBrush` for the Background property to specify a range of colors to apply the gradient background for the AvatarView, as shown in the following code example.
```
Comment (Chrispineagunja) [inline winui/Avatar-view/customization.md:103]: Use the `LinearGradientBrush` or `RadialGradientBrush` for the Background property to specify a range of colors to apply the gradient background for the AvatarView as shown in the following code example.

File: winui/Avatar-view/customization.md [hunk 14]
```diff
@@ -1,10 +1,176 @@
...
+};
+
+{% endhighlight %}
+{% endtabs %}
+
+![WinUI AvatarView control with Gradient Background](avatarview_images/winui_avatarview_gradient.png)
+
+## Font
+
+You can customize the font of initials in the `AvatarView` control by using the `FontFamily` and `Foreground` properties.
```
Comment (Chrispineagunja) [inline winui/Avatar-view/customization.md:149]: Customize the font of initials in the `AvatarView` control by using the `FontFamily` and `Foreground` properties.

**Pull request merged**